app/pods/publications/publication/controller.js

```diff
@@ -396,7 +395,12 @@ export default class PublicationController extends Controller {
       this.showConfirmWithdraw = true;
     } else {
       const publicationStatus = await this.store.findRecord('publication-status', event.id);
-      this.model.publicationFlow.set('status', publicationStatus);
+      if (event.id === CONFIG.publicationStatusPublished.id) {
```

Comment:
suggestie: Aangezien we publicationStatus hier hebben, kunnen we daarvan het id vergelijken ipv terug event te gebruiken?

:heavy_plus_sign:  deze waarde staat niet (meer?) in CONFIG. Moet nog aangepast worden
`PUBLICATION_STATUSES.published.id`

Reply:
Is van een vorige Pull request die toen nog niet op dev stond. Ik fix het :)